### DISCUSSION TITLE
KILLCURLY ADD MORE THEN 1 ACCOUNT METHOD

### DISCUSSION THREAD

**Discussion by videoconsole** [3kh0/ext-remover#651] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
so i found out how you can add more then 1 account using killcurly

How this works: idk 

what do you need: 
1. chrome://settings/signOut not blocked
2.a chrome book that doesnt have this patched

How to do this:
1. Use killcurly to login with a nonschool account like your main email
2.go back to chrome://settings/signOut and sign out (this doesnt sign you out of your main email idk why) 
3. go to the sign in website and sign in with your school account 
4.and done you can now switch between your school and main email the only down side is your main email is set to default for some reason

i am smart

**Comment by Hilololsusy** [comment]
Killcurry doesn't even work

**Reply by videoconsole** [reply]
well it does for me?

**Reply by SpiralSky** [reply]
Its patched in later versions, you can downgrade your chromeos version. sorry for necro

**Comment by FearedFusionX** [comment]
Even after killing GoGuardian, the chrome://signOut url is blocked by the administrator 

bruh

**Reply by Danieltjy1186** [reply]
do you have only goguardian?

**Comment by Mirkwood815** [comment]
Anyway to remove the private account?

**Reply by Danieltjy1186** [reply]
just sign out again.

**Comment by ghost** [comment]
if you want to add it, then make a pull request.
